server/openapi.yml

```diff
@@ -633,48 +533,73 @@ paths:
   /sign_up:
     post:
       tags:
-        - admin
+        - user
       summary: サインアップ
       description: サインアップする
       operationId: signUp
       security: []
       requestBody:
         $ref: "#/components/requestBodies/SignUpBody"
       responses:
-        204:
-          $ref: "#/components/responses/NoContent"
-        400:
-          $ref: "#/components/responses/BadRequest"
+        200:
+          description: サインアップに成功した時のレスポンス
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/User"
+          headers:
+            access-token:
+              $ref: "#/components/headers/access-token"
+            uid:
+              $ref: "#/components/headers/uid"
+            client:
+              $ref: "#/components/headers/client"
```

Comment:
[Q]
expiryはレスポンスとして返さない予定？
トークンの有効期限切れてたらAPIをそもそも叩かないようにしても良さそう（有効期限あるサービスが多いはず）。

https://devise-token-auth.gitbook.io/devise-token-auth/usage/controller_methods#authentication-headers-example
> The date at which the current session will expire. This can be used by clients to invalidate expired tokens without the need for an API request.

Reply:
返す予定やったけど忘れてたわ
追加しとく